bot/exts/moderation/infraction/_utils.py

```diff
@@ -298,3 +300,26 @@ async def send_private_embed(user: MemberOrUser, embed: discord.Embed) -> bool:
             "The user either could not be retrieved or probably disabled their DMs."
         )
         return False
+
+
+class StaffBanConfirmationView(interactions.ViewWithUserAndRoleCheck):
+    """The confirmation view that is sent when a moderator attempts to ban a staff member."""
```

Comment:
This view isn't inherently only for bans - it looks like it could work with any infraction if we wanted it to. I think this name could be more generic.

Also, should this be moved to its own module (e.g. a module for all-things views) - are we putting too much stuff into "utils"?